File: tests/java/example2.java
public class Main {

    // Fonction pour transformer le texte
    public static String TransformText(String text) {
        String text = text.toUpperCase(); // Convertir en majuscules
        String prefix = "Prefix: ";
        return AddPrefix(modifiedText, prefix);
    }

    // Fonction pour ajouter un préfixe
    public static String AddPrefix(String text, String prefix) {
        return prefix + text;
    }

    // Fonction de test
    public static void test() {
        String inputText = "Hello, World!";
        String result = TransformText(inputText);
        System.out.println(result);
    }

    // Fonction principale
    public static void main(String[] args) {
        test();
    }
}

File: tests/rs/example2.rs
// Fonction pour transformer le texte
fn transform_text(text: &str) -> String {
    let text = text.to_uppercase(); // Convertir en majuscules
    let prefix = "Prefix: ";
    add_prefix(&modified_text, prefix)
}

// Fonction pour ajouter un préfixe
fn add_prefix(text: &str, prefix: &str) -> String {
    format!("{}{}", prefix, text)
}

// Fonction de test
fn test() {
    let input_text = "Hello, World!";
    let result = transform_text(input_text);
    println!("{}", result);
}

// Fonction principale
fn main() {
    test();
}
